examples/north_sea/model_config.py

```diff
@@ -97,15 +97,15 @@ def construct_solver(spinup=False, store_station_time_series=True, **model_optio
     coriolis_2d.interpolate(2 * omega * sin(lat * pi / 180.0))
 
     # Setup temporal discretisation
-    default_start_date = datetime.datetime(2022, 1, 1, tzinfo=sim_tz)
-    default_end_date = datetime.datetime(2022, 1, 2, tzinfo=sim_tz)
+    default_start_date = datetime.datetime(2022, 1, 1, 0, 0, tzinfo=sim_tz)
+    default_end_date = datetime.datetime(2022, 1, 2, 0, 0, tzinfo=sim_tz)
     start_date = model_options.pop("start_date", default_start_date)
     end_date = model_options.pop("end_date", default_end_date)
     dt = 3600.0
     t_export = 3600.0
     t_end = (end_date - start_date).total_seconds()
     if os.getenv("THETIS_REGRESSION_TEST") is not None:
-        t_end = 5 * t_export
+        end_date = datetime.datetime(2022, 1, 1, 0, 5, tzinfo=sim_tz)
```

Comment:
That's 5 minutes past midnight, no? but the timestep is in hours?

Reply:
Good spot! Fixed in https://github.com/thetisproject/thetis/pull/325/commits/1b073425f37bb9ba501a3a3fa8a3471257ddf1ac.